spec/plugins/lapacke/lapacke_spec.rb

```diff
@@ -300,4 +300,39 @@
       end
     end
   end
+  
+  context "math#norm" do
```

Comment:
You don't need to duplicate the tests. You can just put them in math_spec.rb and for the tests that depend on LAPACK (which I think is just the 2-norm), you can rescue the NotImplementedError.